assets/js/api-fetch-shim.js

```diff
@@ -3,9 +3,7 @@
  */
 import apiFetch from '@wordpress/api-fetch__non-shim';
 
-if ( global._apiFetch === undefined ) {
-	global._apiFetch = apiFetch;
-
+if ( global._apiFetchRootURL || global._apiFetchNonceMiddleware ) {
```

Comment:
Can we not source these from `global._googlesitekitBase` (would need to be added of course) rather than generic globals?

Reply:
Indeed! I just haven't got around to it yet. I'll do that ASAP 👍